src/server/saved_objects/schema/schema.ts

```diff
@@ -31,6 +32,14 @@ export class SavedObjectsSchema {
     this.definition = schemaDefinition;
   }
 
+  public isHiddenType(type: string) {
+    if (this.definition && this.definition.hasOwnProperty(type)) {
+      return !!this.definition[type].hidden;
```

Comment:
super nit: using `Boolean(this.definition[type].hidden)` is more consistent with `isNamespaceAgnostic`

Reply:
I am fine switching for readability and being more explicit.